src/platform-includes/getting-started-config/javascript.react.mdx

```diff
@@ -2,12 +2,11 @@
 import React from "react";
```

Comment:
@AbhiPrasad since React 18, there's a updated method of importing react.

I took this from the [official react website ](https://react.dev/learn/add-react-to-an-existing-project)
```javascript
import { createRoot } from 'react-dom/client';

// Clear the existing HTML content
document.body.innerHTML = '<div id="app"></div>';

// Render your React component instead
const root = createRoot(document.getElementById('app'));
root.render(<h1>Hello, world</h1>);
```